arm/Microsoft.ApiManagement/service/.bicep/nested_authorizationServers.bicep

```
@@ -100,10 +100,10 @@ module authorizationServer '../authorizationServers/deploy.bicep' = {
 }
 
 @description('The name of the API management service authorization server')
-output authorizationServerName string = authorizationServer.outputs.authorizationServerName
+output name string = authorizationServer.outputs.authorizationServerName
```

Since we also changed the correspondent output names in /authorizationServers/deploy.bicep it should probably be

```suggestion
output name string = authorizationServer.outputs.name
```